typescript/README.md

```diff
@@ -1 +1,178 @@
 # TypeScript Best Practices
+
+1. **Consistent Coding Style**:
+   - **Explanation**: Use a linter like ESLint to maintain a consistent coding style.
+      - @typescript-eslint/parser
+      - @typescript-eslint/eslint-plugin
+      - eslint-config-airbnb-typescript
+
+   Configure ESLint: Create or modify the `.eslintrc.json` file at the root of your project with the following configuration:
+
+   ```json
```

Comment:
I recommend to not add a configuration code since it can oftenly changes. We can say: 
- Use a linter like ESLint to maintain a consistent coding style.
- The recommended configuration => link to https://typescript-eslint.io/getting-started